src/shared/locales/enUs/translations/labs/index.ts

```diff
@@ -4,6 +4,7 @@ export default {
     filterTitle: 'Filter by status',
     search: 'Search labs',
     successfullyUpdated: 'Successfully updated',
+    successfullyCompleted: 'Succesfully Completed',
```

Comment:
May you correct the spelling for _succes**s**fully_ and use lowercase _c_ in the word _completed_ to keep the message consistent with other messages i.e. your message must be changed to "Successfully completed"

Reply:
okay @blestab

Reply:
done @blestab